Makefile

```diff
@@ -265,7 +265,7 @@ KUBECTL_WS ?= $(LOCALBIN)/kubectl-create-workspace
 # renovate: datasource=github-releases depName=kubernetes-sigs/cluster-api
 CLUSTERCTL_VERSION ?= v1.10.6
 # renovate: datasource=github-releases depName=kcp-dev/kcp
-KCP_VERSION ?= v0.27.1
+KCP_VERSION ?= v0.28.0
```

Comment:
As we updated `github.com/kcp-dev/kcp/sdk` to `v0.28.1`, do we also want to update this tag accordingly?